CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.36.0
+* Feature - structured logs and logfile rollover features [#2311](https://github.com/aws/amazon-ecs-agent/pull/2311) and [#2319](https://github.com/aws/amazon-ecs-agent/pull/2319)
+* Feature - new ECS_LOG_* env vars [#2330](https://github.com/aws/amazon-ecs-agent/pull/2330)
```

Comment:
this can probably be combined with previous one? (e.g. "... configurable via XXX env" if you want to mention them..)

Reply:
done